CMSIS/Core_A/Source/irq_ctrl_gic.c

```diff
@@ -1,11 +1,11 @@
 /**************************************************************************//**
  * @file     irq_ctrl_gic.c
  * @brief    Interrupt controller handling implementation for GIC
- * @version  V1.1.1
- * @date     29. March 2021
+ * @version  V1.1.2
```

Comment:
Shall we go for a minor version increment in this case?
I think we are adding a feature not just fixing a bug, right?
```suggestion
 * @version  V1.2.0
```

Reply:
I’ve updated the version number as suggested.